### PR TITLE
feat: remove old landing page

### PR DESCRIPTION
**Pull request by andy-haynes**
This PR removes the old landing page in favor of redirecting to a landing page. Not a high priority but makes it more convenient to have a Component landing page.

### CODE REVIEW

File: apps/web/pages/index.tsx [hunk 1]
```diff
@@ -1,52 +1,17 @@
 import { useRouter } from 'next/router';
-import { useState } from 'react';
+import { useEffect } from 'react';
 
-const DEFAULT_ROOT_COMPONENT = 'andyh.near/widget/ComponentIdTestRoot';
+const DEFAULT_ROOT_COMPONENT = 'andyh.near/widget/LandingPage';
```
Comment (andy-haynes) [inline apps/web/pages/index.tsx:4]: I'd like to get this injected at build time, what's the ideal way of doing this in Vercel?

**Comment by andy-haynes** [comment]
Initially I tried to consolidate this into a `[[...roots]].tsx` module but was having trouble. Is there a better way to handle a redirect from `/` to another route?

**Comment by mpeterdev** [comment]
> Initially I tried to consolidate this into a `[[...roots]].tsx` module but was having trouble. Is there a better way to handle a redirect from `/` to another route?

Instead of letting index load and doing a redirect as an effect we can set up the redirect in the next config
https://nextjs.org/docs/pages/api-reference/next-config-js/redirects

That being said, I think you were likely on the best track with consolidating into one dynamic route. We should be able to always load that route and fallback to the default root component if no path params are provided while also changing the URL in place. I can look into that

**Comment by mpeterdev** [comment]
created #54

**Comment by mpeterdev** [comment]
can this be closed in favor of #58 ?

**Pull request closed without merging**